CMakeLists.txt

```diff
@@ -141,24 +138,33 @@ endif()
 set(DOC_FILES README.md)
 file(GLOB_RECURSE EXAMPLE_FILES examples "*.pyscn" "*.py" "*.md" "*.psl" "*.pslx" "*.scn" "*.xml")
 
-set(OASES_LIBRARY_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/extlibs/qpOASES-3.2.0/")
-add_subdirectory(${OASES_LIBRARY_DIRECTORY} extlibs/libqpOASES)
-find_package(libqpOASES REQUIRED)
-message("libqpOASES_INCLUDE_DIRS = ${libqpOASES_INCLUDE_DIRS}")
-include_directories(${libqpOASES_INCLUDE_DIRS})
-set(QPOASES_INCLUDE_DIR ${libqpOASES_INCLUDE_DIRS} CACHE INTERNAL "")
-sofa_install_libraries(PATHS ${libqpOASES_LIBRARY})
 
 option(SOFTROBOTSINVERSE_ENABLE_PROXQP "Build proxQP wrapper for inverse problems" OFF)
-if(SOFTROBOTSINVERSE_ENABLE_PROXQP)
-    find_package(proxsuite REQUIRED)
-    message("Sofa.SoftRobots.Inverse: proxsuite FOUND. Will build with proxsuite support.")
+option(SOFTROBOTSINVERSE_ENABLE_QPOASES "Build qpOASES wrapper for inverse problems" ON)
+
+if(NOT SOFTROBOTSINVERSE_ENABLE_PROXQP AND NOT SOFTROBOTSINVERSE_ENABLE_QPOASES)
+    message(FATAL_ERROR "Both proxQP and qpOASES solvers are disabled. Please enable at least one solver.")
```

Comment:
I would prefer having the default solver enabled with a warning message. Something like:
"Both proxQP and qpOASES solvers are disabled. At least one solver should be enabled. Enabling qpOASES.""

Reply:
I don't if it is clear but the current default behavior is to have qpOASES enabled and proxQP disabled.
The CMake options are here to control which solver(s) to be enabled or not. 
To me, we should not behave in contradiction to what CMake options have been set, i.e. these CMake option should enable to control the behavior of CMake in accordance to what the user asked  for, if he passed some specific values to the CMake call.  If he did not pass these options, we still have qpOASES only by default.